### DISCUSSION TITLE
How do I get a self signed local domain be trusted in Cypress?

### DISCUSSION THREAD

**Discussion by SaiMadhav9494** [cypress-io/cypress#20409] [Component Testing]
## Issue:
I have a web app running on https in my local, using mkcert as my CA. It is trusted in all the browsers, but when I visit the app in Cypress's browser, the certificate is not getting validated. I see that Cypress has its own CA and tried some workarounds mentioned in https://github.com/cypress-io/cypress/issues/771. Nothing seemed to work for me as I am hoping to get a better solution.

## Environment:
```
Ubuntu 20.04
Cypress 9.4.1
```
## Chrome:
![image](https://user-images.githubusercontent.com/39062289/156065213-b037d9f9-f5d8-4315-9777-2aeca75235e9.png)

## Cypress:
![image](https://user-images.githubusercontent.com/39062289/156065440-a0f7fa06-6959-4850-85cc-e9203ca365e1.png)

## Code:
```
// test file
/// <reference types="cypress" />
import '@testing-library/cypress';

describe('Header text language change', () => {
    beforeEach(() => {
        cy.visit('https://local.ravenslingshot.com:3002')
    });
})
```
```
// cypress.json
{
    "testFiles": "**/*.spec.tsx",
    "componentFolder": "cypress"
}
```

Also, the local app also gets a cookie from its super domain, but I can't see the cookie being set when opening with Cypress. And inside the local app, there is a check for the cookie and if not found, we redirect the user to that super domain's login page running on https.

Thanks in advance.

**Comment by TimPchelintsev** [comment]
Same issue
